jobs/resources-import-normalized-consumer/index.ts

```diff
@@ -16,19 +16,18 @@
 
 import { ResourceImportProcessorError } from '@tech-matters/hrm-job-errors';
 import fetch from 'node-fetch';
-// import { getSsmParameter } from '@tech-matters/hrm-ssm-cache';
+import { getSsmParameter } from '@tech-matters/hrm-ssm-cache';
 // import { SQS } from 'aws-sdk';
 // eslint-disable-next-line prettier/prettier
 import type { SQSBatchResponse, SQSEvent, SQSRecord } from 'aws-lambda';
 // eslint-disable-next-line prettier/prettier
 import type { ImportRequestBody } from '@tech-matters/hrm-types';
 
-
-const hrmResourcesApiUrl = process.env.hrm_resources_api_url as string;
+const resourcesBaseUrl = process.env.resources_base_url as string;
```

Comment:
Maybe `internalResourcesBaseUrl` if I was being super picky - to make it clear it isn't the normal public URL flex calls

Reply:
Makes sense to me :P